extensions/amp-analytics/0.1/variables.js

```diff
@@ -332,6 +332,15 @@ export class VariableService {
       'SESSION_TIMESTAMP': () =>
         this.getSessionValue_(type, SESSION_VALUES.CREATION_TIMESTAMP),
       'SESSION_COUNT': () => this.getSessionValue_(type, SESSION_VALUES.COUNT),
+      'EVENT_TIMESTAMP': (persist) => {
+        // Defaults to true
+        persist = persist != 'false';
+        return this.getSessionValue_(
+          type,
+          SESSION_VALUES.EVENT_TIMESTAMP,
+          persist
+        );
```

Comment:
The persist behavior is unique to this value key. You don't need to drill down the `persist` boolean as an argument, and you should remove `getSessionEventTimestamp_`.

Also nit: use triple equality `!==` for strings since that prevents type coercion.

```suggestion
        const value = this.getSessionValue_(type, SESSION_VALUES.EVENT_TIMESTAMP);
        // String, defaults to 'true'
        if (persist !== 'false') {
          this.persistSession_(type);
        }
        return value;
```